keyboards/idobo/keymaps/via/rules.mk

```diff
@@ -0,0 +1,2 @@
+BOOTMAGIC_ENABLE = light
```

Comment:
```suggestion
BOOTMAGIC_ENABLE = lite
```

Reply:
I can't believe I made that mistake :D
Thanks! I'll fix it in a bit.